funnel/models/project.py

```diff
@@ -436,6 +436,27 @@ def calendar_weeks(self):
             ]
         }
 
+    @property
+    def current_sessions(self):
+        from ..models import Session
```

Comment:
Move the import to the bottom of the file. Importing within the method has a per-call cost. Annotate with `# isort:skip` to stop isort from moving it up.